openwisp_monitoring/monitoring/configuration.py

```diff
@@ -184,10 +184,22 @@ def _get_access_tech():
         'name': '{name}',
         'key': '{key}',
         'field_name': 'rx_bytes',
-        'related_fields': ['tx_bytes'],
+        'related_fields': ['tx_bytes', 'total_bytes'],
```

Comment:
This change is also not required. We are not making any change to the metric. We are only changing the way we show the collected data to the user. 

```suggestion
        'related_fields': ['tx_bytes'],
```